src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9saktillos/K9SakTilLosAdapterTjeneste.kt

```diff
@@ -153,14 +151,33 @@ class K9SakTilLosAdapterTjeneste(
         TODO()
     }
 
+    private fun oppgaveSkalHaVentestatus(event: BehandlingProsessEventDto): Boolean {
+        val åpneAksjonspunkter = event.aksjonspunktTilstander.filter { aksjonspunktTilstand ->
+            aksjonspunktTilstand.status.erÅpentAksjonspunkt()
+        }
+        val oppgaveFeltVerdiDtos = mutableListOf<OppgaveFeltverdiDto>()
+
+        utledAvventerflagg(event.behandlingSteg, åpneAksjonspunkter = åpneAksjonspunkter, oppgaveFeltverdiDtos = oppgaveFeltVerdiDtos)
+
+        return !oppgaveFeltVerdiDtos.any { it.nøkkel == "avventerSaksbehandler" && it.verdi == "true" }
```

Comment:
Jeg tror denne koden blir bedre hvis du deler den opp i to:
1. Metode for å utlede ventetype (returnerer en enum for ventetype).
2. Metode for å opprette feltverdi ut fra ventetype.

I denne metoden kan du da gjøre en enum-sjekk med resultatet i punkt 1.